android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -678,4 +680,47 @@ public static int getTimeframeFromRadioButtonId(int radioButtonId) {
             return AssetPriceTimeframe.ALL;
         }
     }
+
+    public static String getTimeframeString(int assetPriceTimeframe) {
+        Resources resources = ContextUtils.getApplicationContext().getResources();
+        assert resources != null;
+
+        switch (assetPriceTimeframe) {
+            case AssetPriceTimeframe.LIVE:
+                return resources.getString(R.string.trend_1h_text);
+            case AssetPriceTimeframe.ONE_DAY:
+                return resources.getString(R.string.trend_1d_text);
+            case AssetPriceTimeframe.ONE_WEEK:
+                return resources.getString(R.string.trend_1w_text);
+            case AssetPriceTimeframe.ONE_MONTH:
+                return resources.getString(R.string.trend_1m_text);
+            case AssetPriceTimeframe.THREE_MONTHS:
+                return resources.getString(R.string.trend_3m_text);
+            case AssetPriceTimeframe.ONE_YEAR:
+                return resources.getString(R.string.trend_1y_text);
+            case AssetPriceTimeframe.ALL:
+                return resources.getString(R.string.trend_all_text);
+            default:
+                assert false;
+                return "N/A";
+        }
+    }
+
+    public static ErcToken createEthereumErcToken() {
+        ErcToken eth = new ErcToken();
+        eth.name = "Ethereum";
```

Comment:
Are these temporarily hardcoded values?

Reply:
We want to keep that for now as our `getAllTokens` functionality returns only tokens on Ethereum network, it doesn't return ETH itself. So we need to add it manually in the list of assets for now